docker/config/do.conf

```diff
@@ -1,12 +1,12 @@
 export PROJECT_NAME=${PROJECT_NAME:-"graylog"}
 
 export ELASTICSEARCH_IMAGE=${ELASTICSEARCH_IMAGE:-"elasticsearch"}
-export ELASTICSEARCH_VERSION=${ELASTICSEARCH_VERSION:-"2.1"}
-export ELASTICSEARCH_CONFIG_VOLUME=${ELASTICSEARCH_CONFIG_VOLUME:-"./elasticsearch"}
+export ELASTICSEARCH_VERSION=${ELASTICSEARCH_VERSION:-"2.4.1"}
+export ELASTICSEARCH_CONFIG_VOLUME=${ELASTICSEARCH_CONFIG_VOLUME:-"./config/elasticsearch"}
```

Comment:
Please move the changes in this file and in `./docker/docker-compose.yml` into a separate pull request.